server/services/reportsService.ts

```diff
@@ -22,38 +30,53 @@ export = function createReportsService(
     )
   }
 
-  const isUnallocated = (pd: OffenderDetail) => {
+  private static isUnallocated(pd: OffenderDetail) {
     const results = pd.offenderManagers.some(
       (om) => om.active === true && (om.staff.unallocated === true || om.staff.code.endsWith('U'))
     )
     return results
   }
 
-  const getEligibleLicencesWithAddressOrCasLocation = async (): Promise<Case[]> => {
+  private static getPduDescription(probationDetail: OffenderDetail): string {
+    if (!probationDetail) return ''
+    const activeOffenderManager = probationDetail.offenderManagers.find((om) => om.active)
+    if (!activeOffenderManager) return ''
+    const { description } = activeOffenderManager.probationArea
+    return description || ''
+  }
```

Comment:
I think this can probably be reduced/made a bit more concise by turning it into something like the following

```
private static getPduDescription(probationDetail: OffenderDetail): string {
    const activeOffenderManager = probationDetail?.offenderManagers.find((om) => om.active)
    return activeOffenderManager?.probationArea.description || ''
}
  ```
  
There may be a reason that doesn't work, but by carrying the `null` through to the final return check, we don't need to have as many other explicit returns earlier on for similar `null` checks

Reply:
refactored, yeah not sure why I did not do this, thank you!